e2e/progress-bar.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Should show the progress bar showing the completed percent on  challenges', () => {
+  test('Should enter code in the editor and show the progress bar', async ({
+    page
+  }) => {
+    test.slow();
```

Comment:
We shouldn't need to bump the timeout for this one.

Reply:
removed it .